watchtower/wtdb/client_session.go

```diff
@@ -29,6 +29,15 @@ var (
 	// LastApplied value greater than any allocated sequence number.
 	ErrUnallocatedLastApplied = errors.New("tower echoed last appiled " +
 		"greater than allocated seqnum")
+
+	// ErrNoReservedKeyIndex signals that a client session could not be
+	// created because no session key index was reserved.
+	ErrNoReservedKeyIndex = errors.New("key index not reserved")
```

Comment:
Don't see any new tests for this added though the mock has been updated to return this new value.

Reply:
the mock client db doesn't have any unit tests at this point, they will be added in the follow up client db PR. these should never be hit when testing the client as a block box, which is why the current set of high-level client tests can't/don't trigger it